### PR TITLE
Remove ipywidgets from `matsuyama` lecture

### PR DESCRIPTION
**Pull request by kp992**
Remove ipywidgets from `matsuyama` lecture as it does not work with static pages and hangs the build.

Related to #183

### CODE REVIEW

**Review by mmcky — COMMENTED** [review]
@kp992 I am wondering if we should just add `no-execute` tags to this code and let people run this on their local machine if they want to. We can add a small note above the code.

File: lectures/matsuyama.md [hunk 1]
```diff
@@ -809,11 +817,14 @@ def interact_attraction_basis(ρ=0.2, maxiter=250, npts=250):
     return None
 ```
 
-```{code-cell} python3
+```{code-cell} ipython3
+:class: no-execute
```
Comment (kp992) [inline lectures/matsuyama.md:821]: @mmcky is this correctly tagged? The build hangs again.

Comment (mmcky) [reply lectures/matsuyama.md:821]: @kp992 the correct syntax is

```
:tags: [no-execute]
```

Comment (mmcky) [reply lectures/matsuyama.md:821]: actually `:class: no-execute` should work. 

I think the future is `:tags: [skip-execution]`

Let's leave this as is for now. Thanks @kp992

**Review by mmcky — APPROVED** [review]

**Pull request merged**